packages/rosaenlg-lib/src/LanguageFrench.ts

```diff
@@ -301,6 +305,7 @@ export class LanguageFrench extends LanguageImpl {
 
     // for pronouns, order is static: COD then COI
     const triggeredList = objGroups
+      .filter((objGroup) => objGroup.params?.REPRESENTANT !== 'ref')

```

Comment:
You may also allow to do the contrary, using `REPRESENTANT: 'refexpr'`

Reply:
I believe that it is easier to declare that you don't want pronoun.